internal/consensus/block.go

```diff
@@ -29,8 +29,17 @@ type (
 		signature    []byte
 		hash         *crypto.Uint256
 	}
+
+	// signable is an interface used within consensus package to abstract private key
+	// functionality. This interface is used instead of direct structure usage to be
+	// able to mock private key implementation in unit tests.
+	signable interface {
+		Sign([]byte) ([]byte, error)
+	}
```

Comment:
It's an unexpected consequence of PrivateKey interface simplification, but it's needed only for dbft unit test, hence I still consider that we need this PR. Let me check if we have the same problem on the NeoGo side (although we should not).

Reply:
Nice, no problems with NeoGo, you'll like integration PR.

Reply:
See https://github.com/nspcc-dev/neo-go/pull/3711/commits/8fe8f4f035d5230fc933d88bd8aae850b050a1cf.